core/timelock.go

```diff
@@ -0,0 +1,98 @@
+package core
```

Comment:
can we put this in a separate package? it's a useful function to have, but it's not part of the core protocol as i understand it

Reply:
Yes  I originally did that but for testing purpose it was easier to put thing in core just as a hack "to make it work". This package should even be in kyber not drand.